roles/backup_select_resources/README.md

```diff
@@ -0,0 +1,65 @@
+backup_select_resources
+==================
+
+A role to configure backups for selected resources. The role requires an existing backup vault and plan, and adds selected resources to the provided plan. A set of variables for resource selections is included for use as-is or as examples for modification. This role can be combined with the backup_plan_create role to create or update a backup plan if one does not already exist.
+
+Requirements
+------------
+
+AWS User Account with the following permission:
+
+* backup:CreateBackupSelection
+* backup:DeleteBackupSelection
+* backup:GetBackupPlan
+* backup:GetBackupSelection
+* backup:ListBackupSelections
+* iam:GetRole
+
+Role Variables
+--------------
+
+* **plan_name**: (Required) The name of the backup plan you want to use for the selected resources.
+* **selection_name**: (Required) The display name of the resource selection you want to back up.
+* **selection_resources**: (Required) List of resources selected for backup. Can use wild cards and/or combine with selection options below to precisely restrict resources based on various conditions. See included vars for examples.
+* **selection_excluded_resources**: List of resources to exclude from backup
+* **selection_tags**: List of resource tags selected for backup
```

Comment:
I would probably write something like "If not set, it will default to tag_list_backup_or_prod" for all the cases where it applies. So it is not requires anymore.

Reply:
None of these variables default to anything.